src/Discovery/DiscoveryListView.tsx

```diff
@@ -29,6 +39,10 @@ const DiscoveryListView: React.FunctionComponent<Props> = (props: Props) => {
       locale={{
         emptyText: <Empty image={Empty.PRESENTED_IMAGE_SIMPLE} description='No Studies' />,
       }}
+      onChange={() => {
+        // forcing calling useEffect to update adv search filter height
+        setOnHeightChange(!onHeightChange);
```

Comment:
this is going to cause the component to re-render when data in table has been updated from with the table (like by sorting or filtering on access level), just to trigger the `useEffect` hook

don't know if there is a better way to address it 🤦